Makefile

```diff
@@ -231,7 +231,7 @@ GOLANGCI_LINT = $(LOCALBIN)/golangci-lint-$(GOLANGCI_LINT_VERSION)
 
 ## Tool Versions
 KUSTOMIZE_VERSION ?= v5.3.0
-CONTROLLER_TOOLS_VERSION ?= v0.14.0
+CONTROLLER_TOOLS_VERSION ?= v0.16.5
```

Comment:
The version of `controller-gen` is exposed in https://github.com/openshift/sandboxed-containers-operator/blob/devel/config/crd/bases/kataconfiguration.openshift.io_kataconfigs.yaml#L6.
You need to run `make manifests` to regenerate them and a `make bundle` to propagate the change to the bundle.

Reply:
Just updated